app/services/idv/steps/document_capture_step.rb

```diff
@@ -39,22 +39,29 @@ def handle_document_verification_failure(response)
         failure(response.errors.first, extra)
       end
 
-      def handle_stored_response
-        stored_result = document_capture_session&.load_result
-        if stored_result.present? && stored_result.success?
```

Comment:
Safe to assume we don't need the removed `.present?` check?

Reply:
Yeah, we're checking that in `request_should_use_stored_result?` now